### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by atd** [P2Pvalue/pear2pear#40]
In production, community data fails to load. The console shows:

```
(RangeError)                                                     pear.js:186
 __gwt$exception: <skipped>: Maximum call stack size exceeded

```

I would say this has to do with the bug in tests we could not identify

**Comment by atfornes** [comment]
Solved by:  c4cc512,

Please @atd, confirm that it is fixed.

**Comment by atd** [comment]
Works now for me!

:tada:
